docs/operations/data-freshness-workflow/EVAL.md

```
@@ -0,0 +1,20 @@
+## Eval: atualização e completude dos dados públicos
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)